AIO/src/org/aio/activities/quests/SheepShearer.java

```diff
@@ -0,0 +1,150 @@
+package org.aio.activities.quests;
+
+import org.aio.activities.activity.Activity;
+import org.aio.activities.banking.DepositAllBanking;
+import org.aio.util.MakeAllInterface;
+import org.aio.util.Sleep;
+import org.osbot.rs07.api.map.Area;
+import org.osbot.rs07.api.model.GroundItem;
+import org.osbot.rs07.api.model.NPC;
+import org.osbot.rs07.api.model.RS2Object;
+import org.osbot.rs07.api.ui.Tab;
+
+public class SheepShearer extends QuestActivity {
+
+    private MakeAllInterface makeAllInterface;
+
+    private static final Area FARMER = new Area(3188, 3275, 3190, 3270);
```

Comment:
nit: these three items are areas so maybe `FARMER_AREA`, `SHEEP_AREA`, etc. ?

Reply:
I can change these when I do the refactoring with the helper classes :)